docs/source/project-configuration.mdx

```diff
@@ -0,0 +1,302 @@
+
+**If you are using a schema module**, the operation models will be included in the module, which can be imported by other modules in your project.
+
+**If you are embedding your schema in another target**, the operations will also be included in the generated schema namespace that you include in your application target.
+
+##### Absolute path
+
+To generate your operation models into a single, specified directory, use the [`.absolute(path:)`](./code-generation/codegen-configuration#absolute-operations-output) option. These can then be manually included in your project however you see fit.
+
+If you choose to generate the operation models in to an absolute path, you are you responsible for any module creation in order to link the generated files with the rest of your project. You will need to ensure that any targets which include your operation models link to both your schema types module and the `ApolloAPI` library.
+
+### Supporting other configurations
+
+There are many different possible combinations for a modular architecture and we've only shared examples of the most common configurations. Apollo iOS has many different options to support flexible configurations of the schema types and operation models. Please take the time to familiarize yourself with the [codegen configuration](./code-generation/codegen-configuration) and we're confident you can find a combination to suit your project needs.
+
+## Apollo iOS SDK components
+
+These are the libraries that compose the Apollo iOS package:
+
+<table class="api-ref">
+  <thead>
+    <tr>
+      <th>Description</th>
+      <th>Usage</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+<td colspan="2">
+
+#### `Apollo`
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+The core Apollo client library.
+
+Includes the networking and caching APIs, including `ApolloClient` and `ApolloStore`.
+
+</td>
+<td>
+
+Any targets that need to access these core components directly should be linked against `Apollo`.
+
+</td>
+</tr>
+<tr>
+<td colspan="2">
+
+#### `ApolloAPI`
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+Includes the common components that are used by the generated models for your project.
+
+</td>
+<td>
+
+Any targets that include your generated models should be linked to `ApolloAPI`.
+
+The `Apollo` library has a dependency on this target, so any target that links to `Apollo` does not need to link to  `ApolloAPI` directly.
+
+Because the generated models export the `ApolloAPI` library's interface, targets that consume generated models but do not contain them do not need to link to `ApolloAPI` directly.
+
+</td>
+</tr>
+
+<tr>
+<td colspan="2">
+
+#### `ApolloSQLite`
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+Provides a `NormalizedCache` implementation backed by a `SQLite` database
+
+Use this library if you would like to persist cache data across application lifecycles.
+> See the [`SQLiteNormalizedCache` documentation](./caching/cache-setup#sqlitenormalizedcache) for more information on setting up a persistent SQLite cache.
```

Comment:
The formatting on these *note* sections doesn't look very good. It might be better to have them in their own single column row?

Reply:
It doesn't really bother me personally, but I understand that it's an odd convention. If @rkoron007 or someone from Docs team has a suggestion on how to better format this information, I'm all ears. In the mean time, I'm going to leave it as is, so we can get this published.